src/common/telemetry/src/logging.rs

```diff
@@ -35,15 +35,36 @@ use crate::tracing_sampler::{create_sampler, TracingSampleOptions};
 
 pub const DEFAULT_OTLP_ENDPOINT: &str = "http://localhost:4317";
 
+/// The logging options that used to initialize the logger.
 #[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(default)]
 pub struct LoggingOptions {
+    /// The directory to store log files. If not set, logs will be written to stdout.
     pub dir: String,
+
+    /// The log level that can be one of "trace", "debug", "info", "warn", "error". Default is "info".
     pub level: Option<String>,
+
+    /// The log format that can be one of "json" or "text". Default is "text".
+    pub log_format: LogFormat,
+
+    /// Whether to append logs to stdout. Default is true.
+    pub append_stdout: bool,
+
+    /// Whether to enable tracing with OTLP. Default is false.
     pub enable_otlp_tracing: bool,
+
+    /// The endpoint of OTLP. Default is "http://localhost:4317".
     pub otlp_endpoint: Option<String>,
+
+    /// The tracing sample ratio.
     pub tracing_sample_ratio: Option<TracingSampleOptions>,
-    pub append_stdout: bool,
+}
+
+#[derive(Clone, Debug, Copy, PartialEq, Eq, Serialize, Deserialize)]
+pub enum LogFormat {
+    Json,
+    Text,
```

Comment:
Sugg:
```suggestion
pub enum LogFormat {
    #[serde(rename_all = "snake_case")]
    Json,
    #[serde(rename_all = "snake_case")]
    Text,
```

so that we can write small letter `"test"` in the config file